analyzer/src/test/kotlin/managers/PackageManagerTest.kt

```diff
@@ -40,7 +40,7 @@ class PackageManagerTest : WordSpec({
 
             // The keys in expected and actual maps of definition files are different instances of package manager
             // factories. So to compare values use the package manager names as keys instead.
-            val managedFilesByName = managedFiles.mapKeys { (manager, _) ->
+            val managedFilesByName = managedFiles.mapKeys { (manager) ->
```

Comment:
Is the new representation actually also position based?

Reply:
Yes, properties are assigned "from left to right" in order of declaration, and omitted trailing properties are simply skipped.

Reply:
I'll leave it up to you guys, no strong opinion on that but slight preference to keep as is.